tests/utils/blueprints/blueprint-root.test.js

```diff
@@ -0,0 +1,9 @@
+import { blueprintRoot } from '../../../src/utils/blueprints.js';
+import { assert, test } from '../../helpers/testing.js';
+
+test('utils | blueprints | blueprint-root', function () {
+  assert.strictEqual(
+    blueprintRoot.endsWith('src/blueprints/ember-addon'),
```

Comment:
Given that the value of `blueprintRoot` depends on where the codemod is installed locally, the assertion using `endsWith` may be the strongest assertion that we can make (without leaking the implementation details in the test).